doc/ins_sign_tx.md

```diff
@@ -82,7 +108,7 @@ Initializes signing request.
 |output index |  4 | Big endian |
 
 
-### 3 - Set outputs
+### 4 - Set outputs
```

Comment:
Is there any point in keeping these numbers? Let's just delete them, they do not correspond to tx body keys anymore anyway.